SalesforceCommerceCloud/pwa-kit
# Shopper promotions hooks

# Description
This PR implemented Shopper Promotion hooks. That includes usePromotions and usePromotionsForCampaigns hook in the commerce-react-sdk



# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- commerce-react-sdk/src/hooks/ShopperPromotions/query.tsx
- commerce-react-sdk/src/hooks/ShopperPromotions/query.test.tsx

# How to Test-Drive This PR

- Build your commerce-react-sdk after pulling the branch
- Start up your test project
- Go to `localhost:3000/use-promotions` and `localhost:3000/use-promotions-for-campaign` to see the hooks in actions
- Make sure all the unit tests are passing


# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
